homeassistant/components/sensorpush_cloud/__init__.py

```diff
@@ -0,0 +1,28 @@
+"""The SensorPush Cloud integration."""
+
+from __future__ import annotations
+
+from homeassistant.const import Platform
+from homeassistant.core import HomeAssistant
+
+from .coordinator import SensorPushCloudConfigEntry, SensorPushCloudCoordinator
+
+PLATFORMS: list[Platform] = [Platform.SENSOR]
+
+
+async def async_setup_entry(
+    hass: HomeAssistant, entry: SensorPushCloudConfigEntry
+) -> bool:
+    """Set up SensorPush Cloud from a config entry."""
+    coordinator = SensorPushCloudCoordinator(hass, entry)
+    entry.runtime_data = coordinator
+    await coordinator.async_config_entry_first_refresh()
```

Comment:
Not sure if this is still the case with the 'new' runtime_data, but I think you should swap these lines.

This prevents the integration from storing runtime data when it is unable to initialise.